docs/security.html

```diff
@@ -1248,11 +1248,15 @@ <h3 class="anchor-heading"><a id="security_authz" class="anchor-link"></a><a hre
     Kafka ACLs are defined in the general format of "Principal {P} is [Allowed|Denied] Operation {O} From Host {H} on any Resource {R} matching ResourcePattern {RP}".
     You can read more about the ACL structure in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-11+-+Authorization+Interface">KIP-11</a> and
     resource patterns in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-290%3A+Support+for+Prefixed+ACLs">KIP-290</a>.
-    In order to add, remove, or list ACLs, you can use the Kafka ACL CLI <code>kafka-acls.sh</code>. By default, if no ResourcePatterns match a specific Resource R,
-    then R has no associated ACLs, and therefore no one other than super users is allowed to access R.
-    If you want to change that behavior, you can include the following in server.properties.
+    In order to add, remove, or list ACLs, you can use the Kafka ACL CLI <code>kafka-acls.sh</code>. 
+    <h5>Default Behavior Without ACLs:</h5>
+    <p>If a resource (R) does not have any ACLs defined—that is, if no ACL matches the resource—Kafka will restrict access to that resource. In this situation, only super users are allowed to access it.
```

Comment:
nit: `defined-that` and `resource-Kafka` look like hyphenated words. You perhaps want `&mdash;` instead of `-`, so `resource &mdash; Kafka`. Personally, I would rewrite like this:

`If a resource (R) does not have any ACLs defined, meaning that no ACL matches the resource, Kafka will restrict`...